### DISCUSSION TITLE
Backup and migrate

### DISCUSSION THREAD

**Discussion by husnainali09** [Donkie/Spoolman#529] [General]
How can i keep my spoolman database backed up and how do i migrate between instances.

I have spoolman running on a proxmox LXC and one running on a V2.4 directly via kiauh.

**Comment by Donkie** [comment]
See if the instructions here are sufficient for you. I've yet to add specific instructions regarding Kiauh though. https://github.com/Donkie/Spoolman/wiki/Backup-and-Restore
